packages/clay-color-picker/src/index.tsx

```diff
@@ -149,7 +160,13 @@ const ClayColorPicker: React.FunctionComponent<IProps> = ({
 	value = 'FFFFFF',
 	...otherProps
 }: IProps) => {
-	if (value.indexOf('#') === 0) {
+	const [invalidColor, setInvalidColor] = React.useState(false);
+
+	const colorVal = tinycolor(value);
+
+	const isHex = colorVal.getFormat() === 'hex';
```

Comment:
Maybe just simplify for one line?
```suggestion
	const isHex = tinycolor(value).getFormat() === 'hex';
```